packages/ui-react/src/containers/SiteMetadata/SiteMetadata.tsx

```diff
@@ -0,0 +1,24 @@
+import { Helmet } from 'react-helmet';
+import React from 'react';
+import { useConfigStore } from '@jwp/ott-common/src/stores/ConfigStore';
+import { shallow } from '@jwp/ott-common/src/utils/compare';
+import { useTranslation } from 'react-i18next';
+
+const SiteMetadata = () => {
```

Comment:
Should we move it to components? Doesn't look that this one is used as container.

Reply:
`useConfigStore` is used 😄